### ISSUE TITLE
Right sidebar needs feedback on hover for clickable elements

### ISSUE THREAD

**Issue by dvcrn** [notaapp/talk#293]
Version 0.38.1 (Apple Silicon)

Nice new sidebar! I think it urgently needs feedback on hover to know which elements are clickable and which aren't

<img width="222" alt="Screen Shot 2022-08-16 at 18 18 59" src="https://user-images.githubusercontent.com/688326/184844783-818fdd85-0a94-4d23-b992-b918b3fed132.png">

(or even better, automatically show which header the cursor currently is under ✨)

**Comment by erusev** [comment]
> I think it urgently needs feedback on hover to know which elements are clickable and which aren't

> (or even better, automatically show which header the cursor currently is under ✨)

I agree on both, I'll see if we can fix these for the September release.

Thanks, and I'm glad you like the new sidebar 😊
